webapp/src/main/webapp/vue-app/CallButton/main.js

```diff
@@ -42,4 +42,4 @@ export function init(settings) {
     });
     return comp;
   });
-}
\ No newline at end of file
+}
```

Comment:
This file has no changes , could you remove it from the PR ?

Reply:
done